app/controllers/incidents_controller.rb

```diff
@@ -113,14 +121,17 @@ def set_incident
     end
 
     def incident_params
-      params.require(:incident).permit(
+      permitted_params = [
         :title,
         :private_description,
         :private_wrc_decision,
         :public_summary,
         :tags,
         :digest_worthy,
-        incident_competitions_attributes: [:id, :competition_id, :comments, :_destroy],
-      )
+        :visibility,
+        { incident_competitions_attributes: [:id, :competition_id, :comments, :_destroy] },
+      ]
+
+      params.require(:incident).permit(permitted_params)
```

Comment:
Why are you refactoring this into a separate new line, instead of "just" adding `:visibility` to the existing parameter list?